File: ht11-spring-webflux/ht11-server/src/main/java/ru/otus/kulygin/dto/list/CommentListDto.java
package ru.otus.kulygin.dto.list;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import ru.otus.kulygin.dto.CommentDto;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CommentListDto {

    private List<CommentDto> comments;

}

File: ht11-spring-webflux/ht11-client/src/main/web/src/app/models/list/comment-list-dto.ts
import {CommentDto} from "../comment-dto";

export class CommentListDto {

    private _comments: CommentDto[] = [];

    constructor(comments: CommentDto[]) {
        this._comments = comments;
    }

    get comments(): CommentDto[] {
        return this._comments;
    }

    set comments(value: CommentDto[]) {
        this._comments = value;
    }
}